wagtailio/project_styleguide/templates/patterns/components/author/author.yaml

```diff
@@ -0,0 +1,11 @@
+context:
+  value:
+    author: Meagen Voss
```

Comment:
Nitpick: I know this is in the designs, potentially we should refrain from using real names here.

Reply:
Or, only use puns because that would be more fun